includes/Story_Post_Type.php

```diff
@@ -603,14 +603,15 @@ public function get_editor_settings() {
 					'hasUploadMediaAction'  => $has_upload_media_action,
 				],
 				'api'                   => [
-					'users'        => '/web-stories/v1/users/',
-					'currentUser'  => '/web-stories/v1/users/me/',
-					'stories'      => sprintf( '/web-stories/v1/%s/', $rest_base ),
-					'media'        => '/web-stories/v1/media/',
-					'link'         => '/web-stories/v1/link/',
-					'statusCheck'  => '/web-stories/v1/status-check/',
-					'metaBoxes'    => $this->meta_boxes->get_meta_box_url( (int) $story_id ),
-					'storyLocking' => rest_url( sprintf( '/web-stories/v1/%s/%s/lock', $rest_base, $story_id ) ),
+					'users'         => '/web-stories/v1/users/',
+					'currentUser'   => '/web-stories/v1/users/me/',
+					'stories'       => sprintf( '/web-stories/v1/%s/', $rest_base ),
+					'pageTemplates' => sprintf( '/web-stories/v1/%s/', Page_Template_Post_Type::POST_TYPE_SLUG ),
```

Comment:
@spacedmonkey FYI I've added this here as in case of other routes.

Reply:
Should we add the saved templates to the preloaded URL, too? (When we're adding pagination, currently, it's just loading the 100 first)

Reply:
post type slug and rest base isn't always the same thing. See the line above. You should really get the post type object and get the rest base here.

Reply:
Regarding preloading, normally I would say, unless the REST API requested is needed to render the view a user see when the editor loads, it should be deferred. Why stop the editor from loading if a user may never see the page templates feature?